src/components/LanguageSelector/index.less

```diff
@@ -0,0 +1,9 @@
+.language-select {
+  display: flex;
+  justify-content: center;
+
+  .ant-select {
+    width: 100%;
+    margin: 5%;
```

Comment:
Is `5%` intended here?

Reply:
Yes, this is the margin of the dropdown-menu:
![Auswahl_003](https://user-images.githubusercontent.com/100765498/180772549-c4957ac0-7b58-4dc2-bb67-e513732506b3.png)

Reply:
Yes, but why it's relative?

Reply:
You are right! An absolute margin value makes more sense.